ws-nextjs-app/package.json

```
@@ -11,8 +11,9 @@
     "build": "yarn build:local",
     "dev": "yarn setupDevEnv && next dev -p 7081",
     "start": "next start -p 7081 && NODE_ENV=production",
-    "stop": "lsof -t -i:7081 | xargs kill",
-    "test": "jest --ci --colors",
+    "stop": "../scripts/killApp.sh",
+    "test": "jest --ci --colors --selectProjects='Unit Tests'",
+    "test:integration": "jest --ci --colors --selectProjects='Integration Tests - Canonical' 'Integration Tests - AMP' -u",
```

Let's not update the snapshots in the command - it means that the tests won't fail on CI if there is a change to the snapshots. 
```suggestion
    "test:integration": "jest --ci --colors --selectProjects='Integration Tests - Canonical' 'Integration Tests - AMP'",
```